src/services/queries/Events.js

```diff
@@ -2,17 +2,14 @@ import gql from 'graphql-tag';
 import { eventInfo } from '../fragments';
 
 export const EVENTS = gql`
-  query GetEventsForHome($limit: Int = 4) {
+  ${eventInfo}
+  query GetEventsForHome($limit: Int = 4, $name: String = "%%") {
     events(
       order_by: { date: asc }
-      where: { is_active: { _eq: true } }
       limit: $limit
+      where: { is_active: { _eq: true }, title: { _ilike: $name } }
     ) {
-      id
-      title
-      image_url
-      date
-      type
+      ...eventsFragment
```

Comment:
In the [docs](https://www.apollographql.com/docs/react/data/fragments/) seems like fragments should be PascalCase, please confirm

Reply:
Thanks! will apply